robottelo/config/casts.py

```diff
@@ -70,3 +70,30 @@ class Tuple(List):
     """
     def __call__(self, value):
         return tuple(super(Tuple, self).__call__(value))
+
+
+class Dict(List):
+    """Cast a comma separated list of key=value to a dict.
+
+    :param str value: A comma separated string to cast to a dict.
+    """
+    def __call__(self, value):
+        return dict(v.split('=') for v in super(Dict, self).__call__(value))
```

Comment:
I was thinking thast this works for 
```yml
webdriver_desired_capabilities=browserName=firefox,platform=OS X 10.11,version=44.0,screenResolution=1024x768
```

But if one leaves a blank space between vars it fails.

```yml
webdriver_desired_capabilities=browserName=firefox, platform=OS X 10.11, version=44.0, screenResolution=1024x768
```

To fix that problem should add a **strip()**

```python
return dict(v.strip().split('=') for v in super(Dict, self).__call__(value))
```

But still there is a problem when multiple spaces added

```yml
webdriver_desired_capabilities = browserName = firefox, platform = OS X 10.11, version = 44.0, screenResolution = 1024x768
```

So the solution would be more complex
```python
return {key.strip(): value.strip() for key, value in v.split('=') for v in super(Dict, self).__call__(value)}
```